lib/osf-components/addon/components/node-card/component.ts

```diff
@@ -59,4 +60,9 @@ export default class NodeCard extends Component {
     get analyticsScopePrefix() {
         return this.analyticsScope ? `${this.analyticsScope} - ` : '';
     }
+
+    @computed('readOnly', 'node', 'node.nodeType', 'node.currentUserCanEdit')
+    get showDropdown() {
+        return !this.readOnly && this.node && this.node.nodeType === NodeType.Fork && this.node.currentUserCanEdit;
```

Comment:
Why limit it to forks? Seems like `!this.readOnly && this.node.currentUserCanEdit` is more what we want.

Or is that best saved for a future release?

Reply:
Yes, future release. We *should* add controls relevant for all types, but that's more involved because they will be different depending on type/context. I was trying to scope this as just a bugfix for this release.